dolphinscheduler-dao/src/main/resources/sql/dolphinscheduler_h2.sql

```diff
@@ -2185,3 +2185,64 @@ CREATE TABLE t_ds_listener_event
     KEY         idx_status (post_status),
     KEY         idx_event_sign (sign)
 );
+
+
+-- ----------------------------
+-- Table structure for jdbc registry
+-- ----------------------------
+
+DROP TABLE IF EXISTS `t_ds_jdbc_registry_data`;
+CREATE TABLE `t_ds_jdbc_registry_data`
+(
+    `id`               bigint(11) NOT NULL AUTO_INCREMENT COMMENT 'primary key',
+    `data_key`         varchar(256) NOT NULL COMMENT 'key, like zookeeper node path',
+    `data_value`       text         NOT NULL COMMENT 'data, like zookeeper node value',
+    `data_type`        varchar(64)  NOT NULL COMMENT 'EPHEMERAL, PERSISTENT',
+    `client_id`        bigint       NOT NULL COMMENT 'client id',
+    `create_time`      timestamp    NOT NULL default current_timestamp COMMENT 'create time',
+    `last_update_time` timestamp    NOT NULL default current_timestamp COMMENT 'last update time',
+    PRIMARY KEY (`id`),
+    unique KEY `uk_t_ds_jdbc_registry_dataKey`(`data_key`)
+) ENGINE = InnoDB
+  DEFAULT CHARSET = utf8;
+
+
+DROP TABLE IF EXISTS `t_ds_jdbc_registry_lock`;
+CREATE TABLE `t_ds_jdbc_registry_lock`
+(
+    `id`          bigint(11) NOT NULL AUTO_INCREMENT COMMENT 'primary key',
+    `lock_key`    varchar(256) NOT NULL COMMENT 'lock path',
+    `lock_owner`  varchar(256) NOT NULL COMMENT 'the lock owner, ip_processId',
+    `client_id`   bigint       NOT NULL COMMENT 'client id',
+    `create_time` timestamp    NOT NULL default current_timestamp COMMENT 'create time',
+    PRIMARY KEY (`id`),
+    unique KEY `uk_t_ds_jdbc_registry_lockKey`(`lock_key`)
+) ENGINE = InnoDB
+  DEFAULT CHARSET = utf8;
+
+DROP TABLE IF EXISTS `t_ds_jdbc_registry_client_heartbeat`;
+CREATE TABLE `t_ds_jdbc_registry_client_heartbeat`
```

Comment:
Heartbeat is just an attribute of client, so maybe registry_client is better.
```suggestion
CREATE TABLE `t_ds_jdbc_registry_client`
```

Reply:
The origin table name is t_ds_jdbc_registry_client, but I am worried this will conflict with JdbcRegistryClient which is already exist.

Reply:
I think it's OK because it has t_ds prefix, and what about using `DsJdbcRegistryClient` as entity class name?